site/_data/postsData.js

```diff
@@ -0,0 +1,79 @@
+const yaml = require('js-yaml');
+const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
+const matter = require('gray-matter');
+
+const {defaultLocale} = require('./site.json');
+
+/** @type AuthorsData */
+const authorsData = require('./authorsData.json');
+const authorsYaml =
+  /** @type I18nAuthors */ (
+    /** @type TODO */ yaml.load(
```

Comment:
SO this ends up being needed as the `if (authorKey in authorsYaml)` loop has a typing issue saying `authorsYaml` is an `unknown` type. I can't type it to `I18nAuthors` as that doesn't match the type `yaml.load` returns, so I cast it to an `any` first.

Reply:
You miss my point. The line`/** @​type TODO */ yaml.load(` is _not_ doing anything since its argument is not in parenthesis. The cast to `I18nAuthors` actually works; TS allows downcasting from `unknown` to a real type.

Reply:
I think I'd have to disagree (unless I'm misunderstanding you)...

![image](https://user-images.githubusercontent.com/11811422/131587911-7d558c42-fd39-4549-987a-1f8cc719d60d.png)

Wrapping it with the TODO seems to cast it:
![image](https://user-images.githubusercontent.com/11811422/131587993-2664a942-7d65-4f03-a55c-104b728b789a.png)